docs/customization/configuration.md

```diff
@@ -1,5 +1,19 @@
 # Configuring Your App
 
+## `onisite/settings_local.py`
+=======
```

Comment:
This looks like a duplicated header. I think these two lines can just be dropped. Also the text below should be "Open ONI" rather than "open-oni". Unless I missed somewhere in a past search and replace or in changes since. Just a convention we try to follow in documentation to consistently name the software/project.